augraphy/augmentations/badphotocopy.py

```diff
@@ -43,17 +43,17 @@ class BadPhotoCopy(Augmentation):
     def __init__(
         self,
         mask=None,
-        noise_type=0,
-        noise_side="random",
-        noise_iteration=(1, 1),
-        noise_size=(1, 1),
-        noise_value=(30, 60),
-        noise_sparsity=(0.4, 0.6),
-        noise_concentration=(0.4, 0.6),
-        blur_noise=0,
-        blur_noise_kernel=(5, 5),
-        wave_pattern=0,
-        edge_effect=1,
+        noise_type=None,
```

Comment:
@proofconstruction Right now i'm setting parameter value to `None` during the class initialization and reinitialize those values again in another function when we call it later (line 265 below). Do you know a better or more elegant way to do this?

Reply:
This is not relevant now.